POS-System.Business/Services/CartService.cs

```diff
@@ -0,0 +1,59 @@
+using AutoMapper;
+using POS_System.Business.Dtos;
+using POS_System.Data.Repositories.Interfaces;
+using POS_System.Domain.Entities;
+
+namespace POS_System.Business.Services.Interfaces
+{
+    public class CartService(IUnitOfWork _unitOfWork, IMapper _mapper) : ICartService
+    {
+        public async Task<IEnumerable<GetCartDto>> GetAllAsync(CancellationToken cancellationToken = default)
+        {
+            var carts = await _unitOfWork.CartRepository.GetAllAsync(cancellationToken);
+            var mappedCarts = _mapper.Map<IEnumerable<GetCartDto>>(carts);
+            return mappedCarts;
+        }
+
+        public async Task<GetCartDto> GetByIdAsync(int id, CancellationToken cancellationToken)
+        {
+            var cart = await _unitOfWork.CartRepository.GetByIdAsync(id, cancellationToken);
+            // TODO: add actual exception
+            if (cart == null)
```

Comment:
patogus budas yra vietoj == operatoriaus naudot 'is' operatoriu(atitinkamai != butu is not). velgi, nieko nekeicia sitas